src/ExcelDestinationWriter.cs

```diff
@@ -36,7 +36,17 @@ public ExcelDestinationWriter(string path, string destinationPath, MappingCollec
         private DataSet setForExcel;
         private DataTable tableForExcel;
         private readonly string destinationPath;
-        public Mapping currentMapping { get; set; }
+        private Mapping _currentMapping;
+        private ColumnMappingCollection _currentColumnMappings;
+        public Mapping currentMapping
```

Comment:
I would very much prefer for this to be **C**urrentMapping as its a property value.
I do fail to see why we need this though, and can't just use the existing Mapping property?